### PR TITLE
update docs/state.md to reflect the status quo

### PR DESCRIPTION
**Pull request by mrose17**
Update docs/state.md to reflect the contents of the `ledgerState` structure

auditor: @diracdeltas

### CODE REVIEW

File: docs/state.md [hunk 1]
```diff
@@ -384,14 +384,38 @@ WindowStore
...
+    transactions: [ {
+      viewingId: string,
+      surveyorId: string,
+      contribution: {
+        fiat: {
+          amount: number,
+          currency: string
+        },
+        rates: {
+          currency1: number
```
Comment (diracdeltas) [inline docs/state.md]: is this number `BTC` / `currency1` or ...?

Comment (mrose17) [inline docs/state.md]: actually it is currency1/BTC, e.g., the number of dollars/cents per bitcoin...

Comment (diracdeltas) [inline docs/state.md]: ok i'll add that in a comment for clarity

File: docs/state.md [hunk 2]
```diff
@@ -384,14 +384,38 @@ WindowStore
...
+          ...
+        },
+        satoshis: number,
+        fee: number
+      },
+      submissionStamp: number,
+      submissionId: string,
+      count: number,
+      satoshis: number,
```
Comment (diracdeltas) [inline docs/state.md]: what's the difference between this `satoshis` and the `satoshis` in `contribution` above?

**Pull request merged**